docs/source/install/deb.rst

```diff
@@ -25,8 +32,11 @@ Setup repositories
 
   .. code-block:: bash
 
+    # Pick OS version from: trusty, jessie, wheezy
+    export DISTRO=trusty
     wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
-    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
```

Comment:
If `lsb_release -cs` doesn't work, - I'm ok with existing version :+1: 
Just to remove `unset DISTRO`, looks excessive for a doc.

Hardcoding `trusty` might be good yes, but I think we can have feedback/bugreports about other `jessie`, `wheezy` packages.
Especially considering that we're in Beta.